debugProtocol.json

```diff
@@ -2752,6 +2903,16 @@
 				"endColumn": {
 					"type": "integer",
 					"description": "Optional end column of the range covered by this scope."
+				},
+				"kind": {
+					"type": "string",
+					"description": "Optional hint describing the contents of this scope.",
+					"_enum": [ "arguments", "locals", "registers" ],
+					"enumDescriptions": [
+						"Scope contains method arguments.",
+						"Scope contains local variables.",
+						"Scope contains registers."
```

Comment:
Should we add a comment like "there should be at most one scope containing registers."

Reply:
Not sure - it really depends on the host whether having multiple "registers" scopes is a problem or not.  VS Code would be fine with it, and in VS, I'll log a warning about it and ignore the extras.  André, what do you think?

Reply:
Obviously there isn't strong enforcement of these things. But in my opinion the design of registers-as-a-scope is that there should be just one scope for all registers, and the adapter is free to add as many sub-groups as a 'Variable' under that scope, but it is there should just be a single scope. So it is worth calling out the intended usage.